crates/core/tedge_mapper/src/aws/converter.rs

```diff
@@ -31,7 +34,18 @@ impl AwsConverter {
     }
 
     pub fn in_topic_filter() -> TopicFilter {
-        make_valid_topic_filter_or_panic("tedge/measurements")
+        vec![
+            "tedge/measurements",
+            "tedge/measurements/+",
+            "tedge/health",
+            "tedge/health/+",
+            "tedge/events/+",
+            "tedge/events/+/+",
+            "tedge/alarms/+/+",
+            "tedge/alarms/+/+/+",
+        ]
+        .try_into()
+        .unwrap()
```

Comment:
Replacing `unwrap` with error handling is better.

Reply:
In this specific case, unwrap is okay as all the topics are correct and known at compile time.